projects/convenience.md

```diff
@@ -0,0 +1,74 @@
+# Convenience Project
+
+## Background and description
+
+Convenience as a SIG was initially proposed by Ted Young back in 2021. It
+was postponed to prioritize completion of signal specs and stabilization. A
+document was written at this time describing user research to do and a
+brainstorming session,
+https://docs.google.com/document/d/1bjiw5L4E8FztRtfsy4Nu9A6qSP-BJlK6BBzl6Rea-Ds/edit#heading=h.b7qckyjgu28i
+
+### Current challenges
+
+Currently the various language implementations resolve user convenience
+individually and this can lead to drift between different implementations and
+with the spec. An example would be how to handle marking a code block as
+`untraced` which has implementations in at least Java, Ruby and Javascript, each
+done differently, named differently and with no specification.
```

Comment:
I am not sure if I understand this problem in the context of convenience, can you elaborate on why this is bad in that context?

Reply:
Updated in the text.